charts/studio/templates/configmap.yaml

```
@@ -0,0 +1,11 @@
+{{- if not .Values.global.selfSigneCAConfigMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: studio-config # extract to variable
```

```suggestion
  name: studio # extract to variable
```

I suggest we name it `studio`. Since it's a `ConfigMap`, we already know that it will be used for configuration